virtool/ml/data.py

```diff
@@ -67,7 +67,18 @@ async def list(self) -> MLModelListResult:
                     id=one.id,
                     created_at=one.created_at,
                     description=one.description,
-                    latest_release=None,
+                    latest_release=MLModelRelease(
+                        id=one.releases[0].id,
+                        created_at=one.releases[0].created_at,
+                        download_url=one.releases[0].download_url,
+                        github_url=one.releases[0].github_url,
+                        name=one.releases[0].name,
+                        published_at=one.releases[0].published_at,
+                        ready=one.releases[0].ready,
+                        size=one.releases[0].size,
+                    )
+                    if one.releases
+                    else None,
```

Comment:
I think this can be done in the SQL query instead of pulling all the releases for each model and getting the first.

https://stackoverflow.com/a/66116924